dashboard/test/controllers/plc_enrollment_evaluations_controller_test.rb

```diff
@@ -0,0 +1,100 @@
+    @task3 = create(:professional_learning_task, name: 'Answering with your favorite color', professional_learning_module: @module3)
+    @task4 = create(:professional_learning_task, name: 'Learn some Assyrian geography', professional_learning_module: @module4)
+    @task5 = create(:professional_learning_task, name: 'Get clarification on Swallow Species', professional_learning_module: @module2)
+    @task6 = create(:professional_learning_task, name: 'Not getting your favorite color right', professional_learning_module: @module1)
+
+    @question1 = create(:plc_evaluation_question, question: 'What is your name', professional_learning_course: @course)
+    @question2 = create(:plc_evaluation_question, question: 'What is your quest', professional_learning_course: @course)
+    @question3 = create(:plc_evaluation_question, question: 'What is your favorite color?', professional_learning_course: @course)
+    @question4 = create(:plc_evaluation_question, question: 'What is the capital of Assyria?', professional_learning_course: @course)
+    @question5 = create(:plc_evaluation_question, question: 'What is the airspeed velocity of an unladen swallow', professional_learning_course: @course)
+
+    @answer1_1 = create(:plc_evaluation_answer, answer: 'Sir Lancelot', plc_evaluation_question: @question1, professional_learning_task: @task1)
+    @answer1_2 = create(:plc_evaluation_answer, answer: 'Sir Robin', plc_evaluation_question: @question1, professional_learning_task: @task1)
+    @answer1_3 = create(:plc_evaluation_answer, answer: 'Sir Galahad', plc_evaluation_question: @question1, professional_learning_task: @task1)
+    @answer1_4 = create(:plc_evaluation_answer, answer: 'King Arthur', plc_evaluation_question: @question1, professional_learning_task: @task1)
+    @answer1_5 = create(:plc_evaluation_answer, answer: 'Sir Edgecase', plc_evaluation_question: @question1, professional_learning_task: nil)
+
+    @answer2_1 = create(:plc_evaluation_answer, answer: 'I seek the Grail', plc_evaluation_question: @question2, professional_learning_task: @task2)
+    #Not all answers are guaranteed to have associated tasks
+    @answer2_2 = create(:plc_evaluation_answer, answer: 'I seek something else Grail', plc_evaluation_question: @question2, professional_learning_task: nil)
+
+    @answer3_1 = create(:plc_evaluation_answer, answer: 'Blue', plc_evaluation_question: @question3, professional_learning_task: @task3)
+    @answer3_2 = create(:plc_evaluation_answer, answer: 'Yellow - no, blue', plc_evaluation_question: @question3, professional_learning_task: @task6)
+    @answer3_3= create(:plc_evaluation_answer, answer: 'No preference', plc_evaluation_question: @question3, professional_learning_task: nil)
+
+    @answer4_1 = create(:plc_evaluation_answer, answer: 'I don\t know that', plc_evaluation_question: @question4, professional_learning_task: @task4)
```

Comment:
I doubt this actually matters, but this answer string is `'I don<tab> know that'`